draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -454,7 +436,42 @@ about individual client data.
 
 # Applying Privacy Partioning
 
-TODO: talk about how to think about new protocols that are using partitioning, the models that we derive from looking at the existing ones, how you recognize a protocol that's using this, and what tools do we have to evaluate it
+Applying privacy partitioning to an existing or new system or protocol requires the following steps:
+
+1. Identify the types of information used or exposed in a system or protocol, some of which can be used to identify a user or correlate to other contexts.
+1. Partition different types of user-identifying or correlatable information across contexts to minimize the amount of
+user-identifying or correlatable information in any single context.
```

Comment:
```suggestion
1.  Separate user-identifying information from other user data in different contexts and reduce the amount of
user-identifying or correlatable information in each context to the minimum needed to fulfil the intended function.
```

Reply:
I think this is now resolved based on the change below?

Reply:
No, this is a separate point: "minimising amount of user-identifying or correlatable information in any single context" is a too strong goal. Earlier we say you want to "minimise to the information needed to fulfil a certain function". That's something different.